docs/guides/git-vs-composer-install.md

```diff
@@ -0,0 +1,84 @@
+#Git vs Composer Installation of Magento with MFTF
+
+
+###GitHub Installation
+
+If you are planning on contributing a PR to the Magento 2 codebase, you can download Magento 2 from GitHub. Contribution to the codebase is done using the 'fork and pull' model where contributors maintain their own fork of the repo. This repo is then used to submit a pull request to the base repo.
+
+Install guide: [GitHub Installation](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/dev_install.html)
+
+
+###Composer based Installation
+
+Composer install downloads released packages of Magento 2 from the composer repo [https://repo.magento.com](https://repo.magento.com).
+
+All Magento modules and their MFTF tests are put under `<vendor>` directory for convenience of 3rd party developers. With this setup, you can keep your custom modules separate from the core. You can also develop modules in a separate VCS repository and add them to your `composer.json` which will allow them to be installed into the `vendor` directory.
+
+Install guide: [Composer based Installation](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)
+
+
+### MFTF Installation
+
+After installing your Magento project in either of the above ways, the composer dependency `magento/magento2-functional-testing-framework` allows you to download and install MFTF. MFTF will be embedded in your Magento 2 installation and will cover your project with functional tests.  
+
+If you want to contribute a PR into MFTF codebase, you will need to install MFTF in the [Standalone] mode.
+
+
+###Managing modules - Composer vs GitHub
+
+####Via GitHub:
+
+Cloning Magento 2 git repository is a way of installing when you don't have to worry frequently about matching the codebase with production. Your version control system generally holds and manages your `app/code` folder and you can do manual ad-hoc development here.
+
+####Via composer:
+
+Magento 2 advocates the use of composer for managing modules. When you install a module through composer, it is added to `vendor/<vendor-name>/<module>`
+
+If you are developing your own module or adding MFTF tests to the module, you should not edit `vendor` because a composer update could clobber your changes. Instead, you can override a module under `vendor`, by adding files or cloning your module specific git repo to `app/code/<vendor-name>/<module>`.
+
+If you want to distribute the module and its tests, you can initialize a git repo and create a [composer package](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/package/package_module.html). In this way others will be able to download and install your module and access your tests as a composer package, in their `<vendor>` folder.
+
+
+###MFTF test artifacts location
```

Comment:
I don't think that `test artifacts` is the right term, `artifacts` refers to the result of test runs
Use `test materials` instead

Reply:
changed them to test materials.

Reply:
Oops, two were left out. Fixed them both!